build.config.mjs

```diff
@@ -1,13 +1,29 @@
-import dotenv from 'dotenv';
```

Comment:
Let's use single quotes to be consistent with the rest of the project.

Reply:
Is there a linting/formatting config for this project? I noticed my default Prettier was reformatting things but I don't see a project-specific config for either it or ESLint. I see `prettier-plugin-astro` in the dependencies but not Prettier itself.

I can just turn off format-on-save and try to manually match my formatting if needed.

Reply:
I don't believe there is a liniting/formatting config yet, I'll add an issue to GitHub.